CppCoreGuidelines.md

```diff
@@ -13443,7 +13443,7 @@ Alternatives for users
 This section contains rules for people who need high performance or low-latency.
 That is, these are rules that relate to how to use as little time and as few resources as possible to achieve a task in a predictably short time.
 The rules in this section are more restrictive and intrusive than what is needed for many (most) applications.
-Do not blindly try to follow them in general code: achieving the goals of low latency requires extra work.
+Do not try to unthinkingly follow them in general code: achieving the goals of low latency requires extra work.
```

Comment:
To just replace the adjective, I could only offer "mindlessly". The whole sentence could be changed though, something like: "Apply those rules only if you understand them well: ..."

Reply:
replaced "unthinkingly" follow/trust with "always" follow/trust.

Reply:
"do not always follow them" has a significantly different meaning to "do not blindly follow them".

Naïvely would work.

Reply:
changed to "naïvely" in 2 places.